tests/unit/src/test/scala/tests/FileWatcherLspSuite.scala

```diff
@@ -119,4 +125,51 @@ class FileWatcherLspSuite extends BaseLspSuite("file-watcher") {
       _ = assertIsNotDirectory(workspace.resolve("a/non/existent/one/e"))
     } yield ()
   }
+
+  test("didChangeWatchedFiles-notification") {
```

Comment:
This test seems to fail on Windows, do you know why @tgodzik ?

Reply:
There seem to be issues with the path being sent:
```
2023.05.25 15:16:43 WARN  no build tool detected in workspace 'D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
2023.05.25 15:16:42 INFO  Report absolute-path: Uri: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\buildd.sbt

Error message: Illegal character in authority at index 7: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\buildd.sbt
Error: java.lang.IllegalArgumentException: Illegal character in authority at index 7: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\buildd.sbt

2023.05.25 15:16:42 INFO  Report absolute-path: Uri: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\build.sbt

Error message: Illegal character in authority at index 7: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\build.sbt
Error: java.lang.IllegalArgumentException: Illegal character in authority at index 7: file://D:\a\metals\metals\tests\unit\target\e2e\file-watcher\didChangeWatchedFiles-notification\build.sbt
```
it's not a valid URI

Reply:
Yeah, that checks out, thanks.

Reply:
Looks to be working and ready to merge!